app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1> Tasks</h1>
+<!--<h2>Name: <%#= @task[:name] %> Description: <%#= @task[:description] %> Completed at: <%#= @task[:completed_at] %></h2>-->
+
+<h4>Name: <%= link_to @task[:name], '/tasks/:id' %> Description: <%= link_to @task[:description], '/tasks/:id' %> Completed at: <%=  link_to @task[:completed_at], '/tasks/:id' %></h4>
```

Comment:
Consider putting each link on a new line in the file to increase readability. They will still be inline in the rendered html.